spec/services/checksum_validator_spec.rb

```diff
@@ -7,10 +7,8 @@
   let(:endpoint_name) { "fixture_sr3" }
   let(:endpoint) { Endpoint.find_by(endpoint_name: endpoint_name) }
   let(:object_dir) { "#{endpoint.storage_location}/#{DruidTools::Druid.new(druid).tree.join('/')}" }
-  let(:pres_copy) do
-    po = PreservedObject.find_by(druid: druid)
-    PreservedCopy.find_by(preserved_object: po, endpoint: endpoint)
-  end
+  let(:po) { PreservedObject.find_by(druid: druid) }
```

Comment:
All these seeded `find_by` calls would be better served as factory `create` statements (removing the need for many of them).